lib/fabrik/DirectorInstance.js

```diff
@@ -249,7 +291,7 @@ class DirectorInstance extends BaseInstance {
       .then(() => this.manager.deleteBinding(this.deploymentName, params.binding_id));
   }
 
-  buildSecurityGroupRules() {
+  buildIpRules() {
```

Comment:
This should move out of DirectorInstance to PlatformManager class if we have such an implementation.

Reply:
Please comment on https://github.com/cloudfoundry-incubator/service-fabrik-broker/pull/131#discussion_r167478083